js/stores/appStore.js

```diff
@@ -503,6 +503,26 @@ const handleAppAction = (action) => {
     case AppConstants.APP_SET_DICTIONARY:
       appState = appState.setIn(['dictionary', 'locale'], action.locale)
       break
+    case AppConstants.APP_CLEAR_DATA:
+      if (action.clearDataDetail.get('browserHistory')) {
+        handleAppAction({actionType: AppConstants.APP_CLEAR_SITES_WITHOUT_TAGS})
+      }
+      if (action.clearDataDetail.get('downloadHistory')) {
+        handleAppAction({actionType: AppConstants.APP_CLEAR_COMPLETED_DOWNLOADS})
```

Comment:
this was throwing an error that prevented data from being cleared when there were no downloads; fixed in 034e47619bbdefb94b020b6e2bc994304fadc11a